markdown/__init__.py

```diff
@@ -1,23 +1,43 @@
-"""
-Python Markdown
+# Python Markdown
+
+# A Python implementation of John Gruber's Markdown.
 
-A Python implementation of John Gruber's Markdown.
+# - Documentation: https://python-markdown.github.io/
+# - GitHub: https://github.com/Python-Markdown/markdown/
+# - PyPI: https://pypi.org/project/Markdown/
 
-Documentation: https://python-markdown.github.io/
-GitHub: https://github.com/Python-Markdown/markdown/
-PyPI: https://pypi.org/project/Markdown/
+# Started by Manfred Stienstra (http://www.dwerg.net/).
+# Maintained for a few years by Yuri Takhteyev (http://www.freewisdom.org).
+# Currently maintained by Waylan Limberg (https://github.com/waylan),
+# Dmitry Shachnev (https://github.com/mitya57) and Isaac Muse (https://github.com/facelessuser).
 
-Started by Manfred Stienstra (http://www.dwerg.net/).
-Maintained for a few years by Yuri Takhteyev (http://www.freewisdom.org).
-Currently maintained by Waylan Limberg (https://github.com/waylan),
-Dmitry Shachnev (https://github.com/mitya57) and Isaac Muse (https://github.com/facelessuser).
+# - Copyright 2007-2023 The Python Markdown Project (v. 1.7 and later)
+# - Copyright 2004, 2005, 2006 Yuri Takhteyev (v. 0.2-1.6b)
+# - Copyright 2004 Manfred Stienstra (the original version)
 
-Copyright 2007-2018 The Python Markdown Project (v. 1.7 and later)
-Copyright 2004, 2005, 2006 Yuri Takhteyev (v. 0.2-1.6b)
-Copyright 2004 Manfred Stienstra (the original version)
+# License: BSD (see LICENSE.md for details).
 
-License: BSD (see LICENSE.md for details).
 """
+Python-Markdown provides two public functions ([`markdown.markdown`][] and [`markdown.markdownFromFile`][]) both of which wrap
+the public class [`markdown.Markdown`][]. All submodules support these public functions and class and/or provide extension
```

Comment:
In one case it's `<a><code>`, and in the other it's `<code><a>`.
In one case, mkdocstrings-python sets the title, because it know that the link refers to an object with a longer path, and in the other case, autorefs does not add a title because it does not distinguish between regular references (to any heading within the docs) and object references (obtained thanks to object inventories).
I agree that it would be nice to have the title coming from autorefs too. Will open an issue there.

Reply:
> In one case it's `<a><code>`, and in the other it's `<code><a>`. 

That's what I get for trying to work this out while I'm sick. 🤧 Now that you pointed it out, it's obvious. 🤦‍♂️ Should be relatively easy to fix.